OpenRA.Game/FileFormats/WavLoader.cs

```diff
@@ -33,32 +33,33 @@ public class WavLoader : ISoundLoader
 		public enum WaveType { Pcm = 0x1, ImaAdpcm = 0x11 }
 		public static WaveType Type { get; private set; }
 
+		bool wavDetected;
+
 		bool LoadSound(Stream s)
 		{
-			var type = s.ReadASCII(4);
```

Comment:
I looked it up. http://cgit.freedesktop.org/xdg/shared-mime-info/tree/freedesktop.org.xml.in#n4139 is file ending *.wav and just looking for WAVE or WAV.